settings.go

```diff
@@ -79,22 +152,95 @@ func (s *SettingsService) GetSettings(options ...OptionFunc) (*Settings, *Respon
 // GitLab API docs:
 // https://docs.gitlab.com/ce/api/settings.html#change-application.settings
 type UpdateSettingsOptions struct {
-	DefaultProjectsLimit       *int              `url:"default_projects_limit,omitempty" json:"default_projects_limit,omitempty"`
-	SignupEnabled              *bool             `url:"signup_enabled,omitempty" json:"signup_enabled,omitempty"`
-	SigninEnabled              *bool             `url:"signin_enabled,omitempty" json:"signin_enabled,omitempty"`
-	GravatarEnabled            *bool             `url:"gravatar_enabled,omitempty" json:"gravatar_enabled,omitempty"`
-	SignInText                 *string           `url:"sign_in_text,omitempty" json:"sign_in_text,omitempty"`
-	HomePageURL                *string           `url:"home_page_url,omitempty" json:"home_page_url,omitempty"`
-	DefaultBranchProtection    *int              `url:"default_branch_protection,omitempty" json:"default_branch_protection,omitempty"`
-	TwitterSharingEnabled      *bool             `url:"twitter_sharing_enabled,omitempty" json:"twitter_sharing_enabled,omitempty"`
-	RestrictedVisibilityLevels []VisibilityValue `url:"restricted_visibility_levels,omitempty" json:"restricted_visibility_levels,omitempty"`
-	MaxAttachmentSize          *int              `url:"max_attachment_size,omitempty" json:"max_attachment_size,omitempty"`
-	SessionExpireDelay         *int              `url:"session_expire_delay,omitempty" json:"session_expire_delay,omitempty"`
-	DefaultProjectVisibility   *int              `url:"default_project_visibility,omitempty" json:"default_project_visibility,omitempty"`
-	DefaultSnippetVisibility   *int              `url:"default_snippet_visibility,omitempty" json:"default_snippet_visibility,omitempty"`
-	RestrictedSignupDomains    []string          `url:"restricted_signup_domains,omitempty" json:"restricted_signup_domains,omitempty"`
-	UserOauthApplications      *bool             `url:"user_oauth_applications,omitempty" json:"user_oauth_applications,omitempty"`
-	AfterSignOutPath           *string           `url:"after_sign_out_path,omitempty" json:"after_sign_out_path,omitempty"`
+	AdminNotificationEmail              string            `url:"admin_notification_email,omitempty" json:"admin_notification_email,omitempty"`
```

Comment:
All the types (except for slices) should be pointers.

Reply:
My bad, fixed